kotlinpoet/src/commonMain/kotlin/com/squareup/kotlinpoet/Util.kt

```diff
@@ -15,39 +15,36 @@
  */
 package com.squareup.kotlinpoet
 
-import com.squareup.kotlinpoet.CodeBlock.Companion.isPlaceholder
-import java.util.Collections
+import kotlin.reflect.KClass
 
 internal object NullAppendable : Appendable {
-  override fun append(charSequence: CharSequence) = this
-  override fun append(charSequence: CharSequence, start: Int, end: Int) = this
+  override fun append(charSequence: CharSequence?) = this
```

Comment:
Curious about the reason for changing the parameter type here.

Reply:
Previously, `Appendable` was actually `java.lang.Appendable `in Java, and the type of the parameter in its `append(charSequence: CharSequence)` was the platform type, which could actually be `null`.

In the expected definition of `kotlin.text.Appendable`, these functions are actually `append(value: CharSequence?)`, and when detached from the JVM, `?` becomes mandatory.

Reply:
Gotcha, thanks!